src/main/java/uk/gov/register/resources/PreviewEntriesResource.java

```diff
@@ -0,0 +1,92 @@
+package uk.gov.register.resources;
+
+import com.codahale.metrics.annotation.Timed;
+import io.dropwizard.views.View;
+import uk.gov.register.core.Entry;
+import uk.gov.register.core.RegisterReadOnly;
+import uk.gov.register.views.ViewFactory;
+import uk.gov.register.views.representations.ExtraMediaType;
+
+import javax.inject.Inject;
+import javax.ws.rs.*;
+import javax.ws.rs.core.MediaType;
+import java.util.Collection;
+import java.util.Collections;
+
+@Path("/")
+public class PreviewEntriesResource {
+
+    private static final int START = 0;
+
+    protected final ViewFactory viewFactory;
+    private final RegisterReadOnly register;
+
+    @Inject
+    public PreviewEntriesResource(final ViewFactory viewFactory, final RegisterReadOnly register) {
+        this.viewFactory = viewFactory;
+        this.register = register;
+    }
+
+    @GET
+    @Path("/preview-entries-json")
+    @Produces(ExtraMediaType.TEXT_HTML)
+    @Timed
+    public View jsonPreview() {
+        return this.previewPageView(MediaType.APPLICATION_JSON);
```

Comment:
Again a style point (being consistent with the rest of the codebase), we only use `this` in constructors.